pandas/core/arrays/sparse/array.py

```diff
@@ -231,7 +231,7 @@ class SparseArray(PandasObject, ExtensionArray, ExtensionOpsMixin):
         3. ``data.dtype.fill_value`` if `fill_value` is None and `dtype`
            is not a ``SparseDtype`` and `data` is a ``SparseArray``.
 
-    kind : {'integer', 'block'}, default 'integer'
+    kind : {'int', 'block'}, default 'int'
```

Comment:
Please revert this line

Reply:
Hey, thanks for the reply :)
After reverting the line, an error pops up on the validate_doctring.py test, is that ok?
```
################################################################################
################################## Validation ##################################
################################################################################

4 Errors found:
        Parameter "sparse_index" has no description
        Parameter "index" has no description
        **Parameter "kind" type should use "int" instead of "integer"**
        See Also section not found
```

Reply:
This is a known bug with the validator (see #25205) so just have to ignore for now